content/source/docs/enterprise/private/automating-the-installer.html.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: "enterprise2"
+page_title: "Private Terraform Enterprise Installation (Installer Beta)"
+sidebar_current: "docs-enterprise2-private-install-installer"
+---
+
+# Private Terraform Enterprise Automated Installation (Installer Beta)
+
+The installation of Private Terraform Enterprise can be completely automated, for both online and airgapped modes.  Before invoking the installer you must:
+- prepare `/etc/replicated.conf`
+- prepare an [application settings](#application-settings) file
```

Comment:
The installation of Private Terraform Enterprise can be automated for both online and airgapped installs. There are two steps for automating the install, the first is setting up (Replicated)[https://help.replicated.com/], the platform which runs Terraform Enterprise. The second step is setting up Terraform Enterprise itself. Before starting the install process, you must:
- prepare `/etc/replicated.conf`. This file defines the settings for the Replicated install itself
- prepare an [application settings](#application-settings) file. This file defines the settings for the Terraform Enterprise application.
- copy your license file to the instance
- download the `.airgap` bundle to the instance (Airgapped mode only)